### DISCUSSION TITLE
FCB1010 and swells; min volume to low(?)

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#769] [Help] [answered]
I have an FCB1010 foot controller board. Its swell pedals together with GO are IMHO to effective. e.g. the min volume - swells doors fully closed - is somewhat to low. FCB1010 swell pedals have a limited movement range compared with physical organ, and together with GO's wide volume re-action to swell pedal movements, the result is a very sensitive swell function.

My idea is that by increasing the swell min volume the swell would feel less nervous. But how do I configure this. The MIDI Object Swell setup alone doesn't help me. Anyone have ideas?

EDIT 1: Found the Enclosure object parameter AmpMinimumLevel in the ODF. That might do the trick.

Edit 2: The GrandOrgue Help doesn't state of this parameter is an absolute value or a percentage of the max output volume.
So, is 100 (the max value) equal to the current output volume and zero (the min value) equals to total silence?

//Jens

**Comment by jason-barba** [comment, marked as answer]
Problem solved. I had to edit the ODF with new Enclosure AmpMinLevel value.

**Comment by jason-barba** [comment]
In regard to the discussion in https://github.com/GrandOrgue/grandorgue/discussions/781 and https://sourceforge.net/p/ourorgan/discussion/962125/thread/e79cea8f67/?limit=50 I wonder if the .cmb currently can be used instead of editing the ODF. The two discussions seam to contradict each other regarding the use of .cmb files.

Could I have solved my issue by editing a newly exported .cmb?

**Reply by larspalo** [reply]
You must read things in their right context, and for what they are. We're currently just discussing the direction of the future development of GO.

No, you're not supposed to edit the .cmb files manually.

**Reply by jason-barba** [reply]
Clear enough ;-)
